conanfile_rtc.py

```diff
@@ -0,0 +1,23 @@
+from conans import ConanFile
+
+
+class PROJConan(ConanFile):
+    name = "PROJ"
```

Comment:
@chri7325 Does this need to be lower case?

Reply:
@bweather not the package name. Lowercase is only used in the project so the library name is lowercase to match the coding standards.